src/components/ItemsList.js

```diff
@@ -73,6 +73,24 @@ const ItemsList = () => {
     history.push('/additem');
   };
 
+  const deleteItemFromShoppingList = (itemName) => {
+    const deleteItem = window.confirm(
```

Comment:
Would it be possible to write this within a modal instead of `window.confirm `for styling reason? As far as I know, window boxes cannot be changed in style. Also for the other alert boxes we have. Of course only if we are going to change the style of these boxes. (I would recommend it because these boxes do appear differently in every browser. But of course this is not a must) 😄

Reply:
Thanks for the suggestion @YuriDevAT. We actually started with a modal. I agree  it is not possible to customize window confirmation boxes and alerts. To add to that, personally I find  apps that use alerts quite annoying as well because an alert stops me from using the app until I close it. With a modal, I can switch to another page even without closing it. We went with `window.confirm` primarily because  the wireframe and the AC seems to suggest we should use confirmation box. It was mentioned in the weekly sync as well.

Reply:
I love this discussion. I agree that Modal would be a nice implementation for all the alert boxes. It is a nice "todo" item for the coming weeks when you start styling the app.

Reply:
A modal would look nicer, just make sure it's accesible for everyone :slightly_smiling_face: . Here is a guide on accessible modals in react https://www.yanandcoffee.com/2020/05/26/accessible-modals/.